### PR TITLE
[5788] On a cluster setup, Studio only shows the primary node in the Cluster screen

### PR DESCRIPTION
**Pull request by jvega190**
https://github.com/craftercms/craftercms/issues/5788

### CODE REVIEW

**Review by rart — CHANGES_REQUESTED** [review]
Let's get rid of the resource stuff

**Comment by rart** [comment]
Why was the delete cluster stuff removed?

**Comment by sumerjabri** [comment]
@rart we no longer have that feature

File: ui/app/src/components/ClusterManagement/ClusterManagement.tsx [hunk 1]
```diff
@@ -107,19 +65,17 @@ export function ClusterManagement() {
...
+        <ApiResponseErrorState error={error} />
+      ) : fetching ? (
+        <LoadingState />
+      ) : clusters?.length ? (
+        <ClusterGridUI clusters={clusters} />
+      ) : (
+        <EmptyState
+          title={<FormattedMessage id="clusterGrid.emptyStateMessage" defaultMessage="No Clusters Found" />}
+        />
+      )}
```
Comment (rart) [inline ui/app/src/components/ClusterManagement/ClusterManagement.tsx]: Avoid showing empty state when the component bootstraps. Test appropriately all paths.

```suggestion
      ) : clusters ? (
        clusters.length ? <ClusterGridUI clusters={clusters} /> :         <EmptyState
          title={<FormattedMessage id="clusterGrid.emptyStateMessage" defaultMessage="No Clusters Found" />}
        />

      )}
```

**Review by rart — CHANGES_REQUESTED** [review]

File: ui/app/src/components/ClusterManagement/ClusterManagement.tsx (hunk 1 above)
Comment (jvega190) [reply ui/app/src/components/ClusterManagement/ClusterManagement.tsx]: should it be like
```
error 
   ? ErrorState
   : clusters 
      ? clusters.length 
          ? ClusterGrid
          : EmptyState
      : LoadingState
  ```    
?

**Review by rart — APPROVED** [review]

**Pull request merged**